wandb/sdk/wandb_init.py

```diff
@@ -660,15 +657,15 @@ def init(self) -> Union[Run, RunDisabled, None]:  # noqa: C901
         return run
 
     def _set_init_telemetry_attrs(self, kwargs):
```

Comment:
nit: maybe we can remove this function and just add the values directly to the `_init_telemetry_obj`?

Reply:
i dont love this, but i dislike how cluttered _setup() is already also.  but given the current state, i agree,  i moved it into the setup function.